aws/alb-ingress-iam-policy.json

```diff
@@ -0,0 +1,140 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "acm:DescribeCertificate",
+        "acm:ListCertificates",
+        "acm:GetCertificate"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:AuthorizeSecurityGroupIngress",
+        "ec2:CreateSecurityGroup",
+        "ec2:CreateTags",
+        "ec2:DeleteTags",
+        "ec2:DeleteSecurityGroup",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeAddresses",
+        "ec2:DescribeInstances",
+        "ec2:DescribeInstanceStatus",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeNetworkInterfaces",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeTags",
+        "ec2:DescribeVpcs",
+        "ec2:ModifyInstanceAttribute",
+        "ec2:ModifyNetworkInterfaceAttribute",
+        "ec2:RevokeSecurityGroupIngress"
+      ],
+      "Resource": "*"
```

Comment:
Can we be more specific with this? Particularly here, but on other resources where feasible.

Reply:
In theory yes, we could set the ALB service account ARN in there and it *might* work (not sure considering that same policy file is used by all examples/documentation on aws/eks and nobody seems to change it...).
But the point is that the policy is only associated with the service account running the ALB already so that link is already done, there is no benefit in setting this explicitly here as I see it. (cf `eksctl create iamserviceaccount` command in this doc).

Also looking at that command, I can't set the service account ARN in the policy file as there's a chicken and egg problem by doing so... so nope I guess.